machine-learning/house_price/common/prepare_data.dig

```diff
@@ -0,0 +1,129 @@
+_export:
+  phase: training
+
++shuffle:
+  td>: ../queries/shuffle.sql
+  create_table: ${source}_shuffled
+
++split:
+  _parallel: true
+
+  +training:
+    td>: ../queries/split_train.sql
+    engine: presto
+    create_table: ${source}_${phase}
+
+  +test:
+    td>: ../queries/split_test.sql
+    engine: presto
+    phase: test
+    create_table: ${source}_${phase}
+
+## Enable if min-max scaling required
+# +minmax_preparation:
+#   +train:
+#     td>: ../queries/minmax_train.sql
+#     engine: presto
+#     store_last_results: true
+  
+#   +test:
+#     td>: ../queries/minmax_test.sql
+#     engine: presto
+#     store_last_results: true
+
++vectorize:
+  _parallel: true
+
+  ## Vectorize without feature scaling
+  # +training:
+  #   td>: ../queries/vectorize_features.sql
+  #   target_table: ${source}_${phase}
+  #   create_table: ${phase}
+  
+  # +test:
+  #   td>: ../queries/vectorize_features.sql
+  #   phase: test
+  #   target_table: ${source}_${phase}
+  #   create_table: ${phase}
+
+  ## log1p rescaling
+  +training:
+    td>: ../queries/vectorize_log1p_features.sql
+    target_table: ${source}_${phase}
+    create_table: ${phase}
+  
+  +test:
+    td>: ../queries/vectorize_log1p_features.sql
+    phase: test
```

Comment:
race condition on `phase` (?)
https://console.treasuredata.com/app/workflows/534349/sessions/3871847

Reply:
^ it was due to typo in my testing.